projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -685,6 +689,15 @@ export class IgxOverlayService implements OnDestroy {
                 this._overlayInfos.filter(x => x.settings.closeOnOutsideClick && !x.settings.modal &&
                     x.closeAnimationPlayer &&
                     x.closeAnimationPlayer.hasStarted()).length === 1) {
+
+                // click event is not fired on iOS. To make element "clickable" we are
+                // setting the cursor to pointer
+                if (this.platformUtil.isIOS && !this._cursorStyleIsSet) {
```

Comment:
There could be a unit test, w/ something like `expect(mockDocument.body.style.cursor).toEqual('pointer')`  when `platformUtil.isIOS` is mocked to be `true`.
This will not test the actual integration, but the logic will be covered.
It might be a bit overkill though, up to you if you want to include it.

Reply:
great idea!!!
Will do it now.
Thanks!